manager.py

```diff
@@ -0,0 +1,160 @@
+import docker
+import psycopg2
+import json
+import time
+
+
+def get_env_vars(client):
+    global postgres_user
+    global postgres_pass
+    global postgres_db
+
+    container = client.containers.get("citus_master")
+
+    env_vars = container.exec_run(cmd='env')
+
+    for each_line in env_vars.splitlines():
+        if 'POSTGRES' in each_line:
+            (key, val) = each_line.split("=")
+            if key == "POSTGRES_USER":
+                postgres_user = val
+            elif key == "POSTGRES_PASSWORD":
+                postgres_pass = val
+            elif key == "POSTGRES_DB":
+                postgres_db = val
+
+
+def connect_to_db(client):
+    conn = None
+    a = 0
+
+    get_env_vars(client)
+
+    while a < 10 and conn is None:
```

Comment:
Since we've added the health check, this try loop isn't necessary. I'd just remove this method entirely and have the bare `connect` call in the calling function.